build_docs.pl

```diff
@@ -1069,6 +1074,7 @@ sub usage {
           --repos_cache     Directory to which working repositories are cloned.
                             Defaults to `<script_dir>/.repos`.
           --skiplinkcheck   Omit the step that checks for broken links
+          --warnlinkcheck   Checks for broken links but does not fail if they exist
```

Comment:
We might want to define how things work if both `--skiplinkcheck` and `--warnlinkcheck` are provided. I assume we'd ignore and not warn about anything if we skip link checking entirely. But do we want to raise an error saying "`--warnlinkcheck` cannot be used if `--skiplinkcheck` is provided"?

Reply:
I've add a check to fail if both are used: fdc74aa2d651c62fb9eafa72b7c879489714a1ad